src/main/java/org/team1540/robot2024/commands/indexer/OuttakeCommand.java

```diff
@@ -0,0 +1,30 @@
+package org.team1540.robot2024.commands.indexer;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import org.team1540.robot2024.subsystems.indexer.Indexer;
+
+public class OuttakeCommand extends Command {
```

Comment:
I would not call this outtake command if it's not actually ejecting. Maybe "StageIndexerShot" or something. This is also simple enough to be a `FunctionalCommand` if you want to do that